walletprotocol.py

```diff
@@ -229,6 +230,11 @@ def parse_cmd(self, data):
                 elif command == 'wallet':
                     self.wallet()
 
+                elif command == 'crash':
+                    self.output['message'].write('>>> CRASH requested' + '\r\n')
+                    print("==================CRASH REQUESTED OVER TELNET====================")
+                    raise Exception("CRASHING CRASHING CRASHING CRASHING CRASHING CRASHING CRASHING bail now")

```

Comment:
Although telnet should be removed at some point, adding a crash command looks like an odd idea..
What is the purpose/concept behind proposing a crash command?